modules/platforms/cpp/ignite/common/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ set(PUBLIC_HEADERS
     ignite_time.h
     ignite_timestamp.h
     ignite_type.h
+    mpi.h
```

Comment:
Why is it public?

Reply:
This header is included in the `big_integer.h`, so I suppose we have to include this header in the PUBLIC list. May be I can hide this include through the pimpl or something like this.

Reply:
We should at least move it to `details` folder, as it should not be a public header.